2021-08-09 15:21:32,317 INFO    MainThread:1036798 [wandb_setup.py:_flush():69] setting env: {'entity': 'katiez', 'project': 'cgan_earth', 'api_key': 'be4ee90fd1224a904f7f4764a16a13960cafe300'}
2021-08-09 15:21:32,318 INFO    MainThread:1036798 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-09 15:21:32,319 INFO    MainThread:1036798 [wandb_init.py:_log_setup():337] Logging user logs to /rds/general/user/jz12018/home/cgan_earth/wandb/run-20210809_152132-2o6nerky/logs/debug.log
2021-08-09 15:21:32,319 INFO    MainThread:1036798 [wandb_init.py:_log_setup():338] Logging internal logs to /rds/general/user/jz12018/home/cgan_earth/wandb/run-20210809_152132-2o6nerky/logs/debug-internal.log
2021-08-09 15:21:32,319 INFO    MainThread:1036798 [wandb_init.py:init():370] calling init triggers
2021-08-09 15:21:32,320 INFO    MainThread:1036798 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-08-09 15:21:32,320 INFO    MainThread:1036798 [wandb_init.py:init():419] starting backend
2021-08-09 15:21:32,321 INFO    MainThread:1036798 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-09 15:21:32,383 INFO    MainThread:1036798 [backend.py:ensure_launched():135] starting backend process...
2021-08-09 15:21:32,441 INFO    MainThread:1036798 [backend.py:ensure_launched():140] started backend process with pid: 1036825
2021-08-09 15:21:32,443 INFO    MainThread:1036798 [wandb_init.py:init():424] backend started and connected
2021-08-09 15:21:32,445 INFO    MainThread:1036798 [wandb_init.py:init():472] updated telemetry
2021-08-09 15:21:32,446 INFO    MainThread:1036798 [wandb_init.py:init():491] communicating current version
2021-08-09 15:21:33,945 INFO    MainThread:1036798 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.11.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-09 15:21:33,946 INFO    MainThread:1036798 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-08-09 15:21:34,156 INFO    MainThread:1036798 [wandb_init.py:init():529] starting run threads in backend
2021-08-09 15:21:34,227 INFO    MainThread:1036798 [wandb_run.py:_console_start():1623] atexit reg
2021-08-09 15:21:34,229 INFO    MainThread:1036798 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.REDIRECT
2021-08-09 15:21:34,232 INFO    MainThread:1036798 [wandb_run.py:_redirect():1502] Redirecting console.
2021-08-09 15:21:34,236 INFO    MainThread:1036798 [wandb_run.py:_redirect():1558] Redirects installed.
2021-08-09 15:21:34,236 INFO    MainThread:1036798 [wandb_init.py:init():554] run started, returning control to user process
2021-08-09 15:21:34,239 INFO    MainThread:1036798 [wandb_config.py:__setitem__():141] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x14a624cf2390>>
2021-08-09 15:21:34,239 INFO    MainThread:1036798 [wandb_run.py:_config_callback():872] config_cb no_cuda False None
2021-08-09 15:21:34,241 INFO    MainThread:1036798 [wandb_config.py:__setitem__():141] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x14a624cf2390>>
2021-08-09 15:21:34,241 INFO    MainThread:1036798 [wandb_run.py:_config_callback():872] config_cb seed 42 None
2021-08-09 15:21:34,242 INFO    MainThread:1036798 [wandb_config.py:__setitem__():141] config set log_interval = 100 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x14a624cf2390>>
2021-08-09 15:21:34,242 INFO    MainThread:1036798 [wandb_run.py:_config_callback():872] config_cb log_interval 100 None
2021-08-09 15:21:34,243 INFO    MainThread:1036798 [wandb_watch.py:watch():40] Watching
2021-08-09 15:21:34,246 INFO    MainThread:1036798 [wandb_watch.py:watch():40] Watching
2021-08-09 15:21:34,272 INFO    MainThread:1036798 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 1
2021-08-09 15:21:34,276 INFO    MainThread:1036798 [wandb_run.py:_restore():1565] restore
2021-08-09 15:21:36,754 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2021-08-09 15:21:37,003 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2021-08-09 15:21:37,191 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 3545
}

2021-08-09 15:21:37,294 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 3545
}

2021-08-09 15:21:37,398 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3545
  total_bytes: 3545
}

2021-08-09 15:21:37,502 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3545
  total_bytes: 3545
}

2021-08-09 15:21:37,605 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3545
  total_bytes: 3545
}

2021-08-09 15:21:37,709 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3545
  total_bytes: 3545
}

2021-08-09 15:21:37,812 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3545
  total_bytes: 3545
}

2021-08-09 15:21:38,177 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3545
  total_bytes: 3545
}

2021-08-09 15:21:38,281 INFO    MainThread:1036798 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3545
  total_bytes: 3545
}

2021-08-09 15:21:39,430 INFO    MainThread:1036798 [wandb_run.py:_show_files():1937] logging synced files
